.github/workflows/build-python-wheels.yaml

```diff
@@ -87,10 +87,12 @@ jobs:
         env:
           ARCHFLAGS: "-arch x86_64 -arch arm64"
         run: python3 setup.py -v bdist_wheel
+      - name: Display structure of macOS files
```

Comment:
Did you mean to keep this in the workflow?

Reply:
Oh yeah I'll remove those actually.